spec/VehicleSignalSpecification.vspec

```diff
@@ -166,6 +166,17 @@ Vehicle.Driver:
 
 #include Driver/Driver.vspec Vehicle.Driver
 
+#
+# Occupant Data
+#
+
+Vehicle.Occupant:
+  type: branch
+  instances:
+    - Row[1,2]
+    - ["DriverSide","Middle","PassengerSide"]
```

Comment:
I am still concerned about these kinds of instances. For one, it does not match the current definition of Seat, which has Pos[1,3]. It is not clear to me that we can always define a driver position. For example, if the driver is in the middle of the vehicle, with a passenger on each side of them, what would we use for occupant position? A count such as Pos[1,3] seems better to represent this, and a reference (such as what I propose above) would be even better, I think.

Reply:
It actually match the current seat definition, see https://github.com/COVESA/vehicle_signal_specification/blob/master/spec/Cabin/Cabin.vspec#L145

```
Seat:
  type: branch
  instances:
    - Row[1,2]
    - ["DriverSide","Middle","PassengerSide"]
  description: All seats.
#include SingleSeat.vspec Seat
```

The use of Left/Right vs Driver/Passenger-side side was heavily discussed some time ago when we change to Driver/Passenger-side, and it comes up to the surface now and then, like in https://github.com/COVESA/vehicle_signal_specification/pull/718. We have actually some coverage for the specific use-case of a driver in middle-seat in https://github.com/COVESA/vehicle_signal_specification/blob/master/spec/Cabin/Cabin.vspec#L156, then left is considered as driver side. 

In my latest commit I just reused the current instance declaration for seats (as well as doors and windows). IMHO having the same type of instances for occupants as for seats sort of make sense